shell/edit/management.cattle.io.project.vue

```diff
@@ -103,8 +104,11 @@ export default {
   methods: {
     async save(saveCb) {
       try {
-        // clear up of the unused resourceQuotas will now be done on the model side
+        if (this.errors) {
+          clear(this.errors);
```

Comment:
`clear` works by maintaining the reference right, which i don't think we need? So `this.errors = []` is the cleanest and quickest solution. for details see https://jsben.ch/hyj65 (new init vs splice)

though.... we're not calling this often or on a big array and the difference isn't that much. up to you whether to change this now.

Reply:
I think that I misinterpreted your previous comment on the topic[^1]. I originally implemented this with `this.errors = []` and can easily revert it back. 

[^1]: https://github.com/rancher/dashboard/pull/10749#discussion_r1551368844

Reply:
Gah, i am 100% wrong. could have sworn it was `= []`. Please raise my memory as something that needs improving in our retro